recipe/doc/recipe_guide.md

```diff
@@ -0,0 +1,87 @@
+If your library is properly configured as such, you will need to do a bit of extra work to ensure that your conda package supports this as well:
+
+You must [ignore the run exports](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements) of any CUDA packages that your package depends on.
+Otherwise, the `run_exports` would require the runtime libraries to have versions equal to or greater than the versions used to build the package.
+You must add explicit runtime dependencies (or `run_constrained` for soft/optional dependencies) that specify the desired version range for the dependencies whose run exports have been ignored. 
+
+As an example, consider that you have built a package that requires libcublas:
+
+```
+requirements:
+  build:
+    - compiler(‘cuda’)
+  host:
+    - libcublas
+    - cuda-version=12.4
+```
+
+By default, at runtime your library will require having the libcublas version corresponding to CUDA 12.4 or newer.
+To make this compatible with all CUDA 12 minor versions 12.0+, you must add the following:
+```
+build:
+  # Ignore run exports in your build section
+  ignore_run_exports_from:
+    - {{ compiler('cuda') }}
+    - libcurand-dev
+
+requirements:
+  run:
+    # Since we’ve ignored the run export, pin manually, but set the min to just “x” since we support any libcublas within the same major release, including older versions
+    - pin_compatible(libcublas, min_pin=”x”, max_pin=”x”)
+```
+
+
+## Cross-compilation
+
+The CUDA recipes are designed to support cross-compilation.
+As such, a number of CUDA components on conda-forge are split into `noarch: generic` component packages that are named according to the supported architecture, rather than being architecture-specific packages.
+The canonical example is [the cuda-nvcc package](https://github.com/conda-forge/cuda-nvcc-feedstock/blob/main/recipe/meta.yaml) that contains the CUDA `nvcc` compiler.
+This package is split into the `cuda-nvcc` package – which is architecture specific and must be installed on the appropriate target platform (e.g.
+x86-64 Linux) – and the `cuda-nvcc-${TARGET_PLATFORM}` packages – each of which is architecture-independent and may be installed on any target, but are only suitable for use in compiling code for the specified target platform.
+This approach allows using host machines with a single platform to compile code for multiple platforms.
+
+
+## Directory structure
+
+### Linux
+
+The conda-forge CUDA packages aim to satisfy two sets of constraints.
+On one hand, the packages aim to retain as similar a structure as possible to the CUDA packages that may be installed via system package manager (e.g.
+apt).
+On the other hand, the packages aim to provide a seamless experience at both build-time and runtime within conda environments.
+To satisfy the first requirement, all files in CUDA conda packages are installed into the `$PREFIX/targets` directory.
```

Comment:
Should we clarify that this is really something like `$PREFIX/targets/$TARGET_PLATFORM/`?